init_q_x: 0.51
init_q_y: 0.511
init_q_z: -0.049
init_q_w: 0.489